frontend/src/components/table.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import '../styles/table.css';
+
+interface driversSettings {
```

Comment:
Change the name of this interface to "Driver". This name is much simpler and will make the reading of your functions' types much easier.